src/liballoc/heap.rs

```diff
@@ -395,6 +425,12 @@ mod imp {
     }
 
     pub fn stats_print() {}
```

Comment:
I think it's fine to leave this as-is, I don't think it's really worth adding an implementation unless it's easy (this is very rarely called)